### PR TITLE
feat: volatility targets, CZAR loss, and testnet topic examples

### PR DESCRIPTION
**Pull request by jefferythewind**
Core library changes:
- workflow.py: Add target_type parameter (log_return | volatility) and compute_volatility_target_polars() for std of 1-min log returns
- czar_loss.py (new): CZAR directional loss with gradient/hessian for custom LightGBM training — penalizes wrong-sign predictions, softens near-zero returns, normalizes by local volatility
- __init__.py: Export czar_loss, czar_gradient, czar_hessian, make_czar_objective

Tests:
- test_volatility_target.py: 8 tests for volatility target computation

Notebooks (example/illustration code):
- Reorganize all topic scripts under notebooks/testnet/topic_*/
- Add example scripts for all testnet topics: 38, 41, 42 (8h price), 57, 83, 84 (8h log-return), 61, 62, 63 (24h log-return), 71 (NEAR 8h), 79-82 (15m volatility)
- Add CZAR V1 model scripts for 8h price topics (38, 41, 42)
- Add dashboard.sh convenience script
- Remove notebooks/shared/ — keep deploy scripts at top level

Docs:
- README: Add topic reference tables, volatility workflow example
- AGENTS.md: Fix paths for testnet/ subfolder
- .gitignore: Add **/runs/ for training artifacts

---
## Summary by cubic
Adds volatility targets (with √horizon scaling) and a CZAR directional loss for `LightGBM`, enabling sign-aware training and correct outputs for realised volatility topics. Consolidates volatility topic scripts into a single grid-retrain per topic, vectorizes vol features (~900× faster), adds an ETH 4h vol example (Topic 85), and updates docs (README adds volatility topics and SOL log-return testnet ID).

- **New Features**
  - `AlloraMLWorkflow`: `target_type` ("log_return" | "volatility"); volatility = std of 1‑min log returns over horizon, scaled by √H; default remains "log_return".
  - `czar_loss.py`: CZAR loss with gradient/hessian and `make_czar_objective`; exported via `allora_forge_builder_kit/__init__.py`.
  - Volatility topics: one `model_grid_retrain.py` per topic (79–82) with vectorized features; Topic 85 (ETH 4h vol) grid retrain and importance-group scripts; `notebooks/dashboard.sh`; `.gitignore` ignores `**/runs/`.
  - Docs & examples: README adds volatility topics and workflow; new example/model scripts for topics 38, 41, 42, 57, 61–63, 71, 83–85; SKILL/AGENTS updates.

- **Bug Fixes**
  - Correct volatility target magnitude by multiplying rolling std by √`target_bars` to match reputer ground truth; tests added.
  - Docs: update Testnet ID for SOL/USD log-return topic.

<sup>Written for commit d8513f6b4cf405e713f3c469106476d063d8a0b9. Summary will update on new commits.</sup>

<a href="https://cubic.dev/pr/allora-network/allora-forge-builder-kit/pull/31?utm_source=github" target="_blank" rel="noopener noreferrer" data-no-image-dialog="true"><picture><source media="(prefers-color-scheme: dark)" srcset="https://www.cubic.dev/buttons/review-in-cubic-dark.svg"><source media="(prefers-color-scheme: light)" srcset="https://www.cubic.dev/buttons/review-in-cubic-light.svg">Review in cubic</picture></a>

### CODE REVIEW

**Comment by jefferythewind** [comment]
closing, superseded by #42
